### PR TITLE
Update configuration for pipeline updates

### PR DESCRIPTION
**Pull request by crissupb**
Configuration changes to support deployment pipeline.

### CODE REVIEW

File: source/buildspec.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+version: 0.2
+
+env:
+    variables:
+      ARTIFACT_BUCKET: "login-gov.lambda-functions.894947205914-us-west-2"
```
Comment (zachmargolis) [inline source/buildspec.yml:5]: is this a real bucket ID or an example? if it's real I think we need to remove it

I can go put it in CircleCI secrets/config if you don't have access

File: source/template.yaml [hunk 2]
```diff
@@ -239,12 +240,38 @@ Resources:
...
           FromPort: 443
           ToPort: 443
-          # Should be another security group or VPC CIDR block
-          CidrIp: 0.0.0.0/0
+          SourceSecurityGroupId: !Sub 
+              - '{{resolve:ssm:/${Environment}/network/securitygroup/idp/id:1}}'
+              - Environment: !Ref environment
+
+  # add all functions that need gitsha alias to this list
+  ApplicationLambdaArns:
```
Comment (zachmargolis) [inline source/template.yaml:248]: Do you think we should try to template this file? There's a lot of copy-paste to add a new lambda

File: source/buildspec.yml (hunk 1 above)
Comment (crissupb) [reply source/buildspec.yml]: It is unused and has been removed.

File: source/buildspec.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    runtime-versions:
+      ruby: 2.7
+    commands:
+      - pip3 install --upgrade aws-sam-cli 
+
+  build:
+    commands:
+      - cd $CODEBUILD_SRC_DIR
+      - GITSHA=$(cat gitsha.txt)
+      - echo $GITSHA
```
Comment (zachmargolis) [inline source/buildspec.yml:14]: do we need to save the `gitsha.txt` to the workspace between jobs?

**Review by zachmargolis — APPROVED** [review]
LGTM

File: source/buildspec.yml (hunk 3 above)
Comment (crissupb) [reply source/buildspec.yml:14]: No.  This is just a way to include the sha in the build artifacts to be used during the deploy.

**Pull request merged**